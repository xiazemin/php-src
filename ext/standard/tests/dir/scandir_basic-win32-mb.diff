010+   string(51) "私はガラスを食べられますfile1.tmp"
010-   string(45) "私はガラスを食べられますfile1.tmp"
012+   string(51) "私はガラスを食べられますfile2.tmp"
012-   string(45) "私はガラスを食べられますfile2.tmp"
014+   string(51) "私はガラスを食べられますfile3.tmp"
014-   string(45) "私はガラスを食べられますfile3.tmp"
020+   string(51) "私はガラスを食べられますfile3.tmp"
020-   string(45) "私はガラスを食べられますfile3.tmp"
022+   string(51) "私はガラスを食べられますfile2.tmp"
022-   string(45) "私はガラスを食べられますfile2.tmp"
024+   string(51) "私はガラスを食べられますfile1.tmp"
024-   string(45) "私はガラスを食べられますfile1.tmp"