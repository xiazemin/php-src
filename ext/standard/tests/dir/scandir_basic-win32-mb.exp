*** Testing scandir() : basic functionality ***

-- scandir() with mandatory arguments --
array(5) {
  [0]=>
  string(1) "."
  [1]=>
  string(2) ".."
  [2]=>
  string(45) "私はガラスを食べられますfile1.tmp"
  [3]=>
  string(45) "私はガラスを食べられますfile2.tmp"
  [4]=>
  string(45) "私はガラスを食べられますfile3.tmp"
}

-- scandir() with all arguments --
array(5) {
  [0]=>
  string(45) "私はガラスを食べられますfile3.tmp"
  [1]=>
  string(45) "私はガラスを食べられますfile2.tmp"
  [2]=>
  string(45) "私はガラスを食べられますfile1.tmp"
  [3]=>
  string(2) ".."
  [4]=>
  string(1) "."
}
===DONE===