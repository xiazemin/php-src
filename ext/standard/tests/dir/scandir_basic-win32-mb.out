*** Testing scandir() : basic functionality ***

-- scandir() with mandatory arguments --
array(5) {
  [0]=>
  string(1) "."
  [1]=>
  string(2) ".."
  [2]=>
  string(51) "私はガラスを食べられますfile1.tmp"
  [3]=>
  string(51) "私はガラスを食べられますfile2.tmp"
  [4]=>
  string(51) "私はガラスを食べられますfile3.tmp"
}

-- scandir() with all arguments --
array(5) {
  [0]=>
  string(51) "私はガラスを食べられますfile3.tmp"
  [1]=>
  string(51) "私はガラスを食べられますfile2.tmp"
  [2]=>
  string(51) "私はガラスを食べられますfile1.tmp"
  [3]=>
  string(2) ".."
  [4]=>
  string(1) "."
}
===DONE===